specification/liftrqumulo/resource-manager/readme.python.md

```diff
@@ -2,15 +2,13 @@
 
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
-
 
 ``` yaml $(python)
 title: QumuloMgmtClient
 azure-arm: true
 license-header: MICROSOFT_MIT_NO_VERSION
-package-name: azure-mgmt-qumulo
-namespace: azure.mgmt.qumulo
+package-name: azure-mgmt-qumulostorage
+namespace: azure.mgmt.qumulostorage
```

Comment:
please don't change the package-name and namespace since it of main branch has been approved by arcboard.

Reply:
Sure, I will update it again. It happened due to conflict resolution with main branch

Reply:
Updated